projects/packages/my-jetpack/README.md

```diff
@@ -0,0 +1,26 @@
+# my-jetpack
+
+WP Admin page with information and configuration shared among all Jetpack stand-alone plugins
+
+## Usage
+
+Every Jetpack plugin must include the My Jetpack package.
+
+Require this package and initialize it:
+
+```PHP
+add_action( 'init', function() {
+	Automattc\Jetpack\My_Jetpack\My_Jetpack::init();
```

Comment:
```suggestion
	Automattic\Jetpack\My_Jetpack\Initializer::init();
```

Reply:
I swear I did it! Thanks for catching that